src/libbluechi/common/protocol.h

```diff
@@ -10,6 +10,12 @@
 #include "time-util.h"
 
 /* Configuration defaults */
+
+/* Enable using remote control for BlueChi via TCP */
+#define CONTROLLER_DEFAULT_USE_TCP "true"
+/* Disable using local control for BlueChi via Unix Domain Socket */
+#define CONTROLLER_DEFAULT_USE_UDS "false"
```

Comment:
Don't we want to have both enabled by default (local for agent running on controller node and TCP for external nodes)?

Reply:
I think it would make sense, yes. Till now we only used TCP as primary method for connecting agents, though.

Reply:
Changed the default to true.